cairo

```diff
@@ -1 +1 @@
-Subproject commit 439da05a031c2eda263c4ce12d0b71d20f38205f
+Subproject commit 679d0472d03177df24ad46058cf389725c2914be
```

Comment:
we need to keep cairo on the alpha 7 commit hash

Reply:
yep, old branch - this is the commit, isn't it? 

81c4eb942dace6849a8067dc43263da99579bef8